Code/GrpAttributes.m

```diff
@@ -187,6 +188,46 @@ elif Get(G,"solvable") and Get(G,"Agroup") then
     end if;
     return false;
 end intrinsic;
+*/
+
+intrinsic monomial(G::LMFDBGrp) -> BoolElt
+    {Determine if a group is monomial}
+    g:=G`MagmaGrp;
+    if not Get(G,"solvable") then
+        return false;
+    elif Get(G, "supersolvable") then
+        return true;
+    elif Get(G,"solvable") and Get(G,"Agroup") then
+        return true;
+    else
+        ct:=CharacterTable(g);
+        maxd := Integers() ! Degree(ct[#ct]); // Crazy that coercion is needed
+        stat:=[false : c in ct];
+        ls:= LowIndexSubgroups(G, maxd);
+        if Type(ls) eq NoneType then
+          return None();
+        else
+          hh:=<z`MagmaSubGrp : z in ls>;
```

Comment:
I think this will fail.  The output of LowIndexSubgroups are records in magma with an attribute subgroup which is what you want.  It is possible some magma group types will produce actual subgroups, but I don't think MagmaSubGrp will ever be defined here.

Reply:
I think it's okay: G is an `LMFDBGrp` and I wrote an intrinsic to produce `LMFDBSubGrps` for it using `LowIndexSubgroups`.